test/integration/integration.cc

```diff
@@ -918,6 +918,23 @@ void BaseIntegrationTest::testNoHost() {
   EXPECT_TRUE(response.find("HTTP/1.1 400 Bad Request\r\n") == 0);
 }
 
+void BaseIntegrationTest::testAbsolutePath() {
```

Comment:
yeah +1 was just going to ask this. Do we have full coverage for all error paths? (Doesn't have to be integration test if you don't want, codec test if fine).